src/Aspire.Dashboard/README.md

```diff
@@ -8,3 +10,26 @@ The dashboard is configured with the following keys:
 - `DOTNET_DASHBOARD_APPLICATION_NAME` specifies the application name to be displayed in the UI. This only applies when no resource service URL is specified. When a resource service exists, the service specifies the application name.
 
 Configuration is obtained through `IConfiguration` so can be provided in several ways, such as via environment variables.
+
+## Auth
+
+The dashboard also supports authentication for its various network endpoints.
+
+To opt-out of authentication, set the following environment variables to `1`:
+
+- `DOTNET_RESOURCE_SERVICE_DISABLE_AUTH`
+- `DOTNET_DASHBOARD_WEB_DISABLE_AUTH`
+- `DOTNET_DASHBOARD_OTLP_DISABLE_AUTH`
```

Comment:
These two configuration keys are not currently in use, though I expect they will be.

@JamesNK are you expecting OTEL auth to be enabled by default with explicit opt-out, as per the resource service and dashboard web app?

Reply:
There is an `DOTNET_DASHBOARD_OTLP_AUTH_MODE` with a setting of None. It defaults to None. An upcoming change for security is to make it not have a default value so people explicitly say none.

I think an enum like what OTLP is doing is better because it can serve double duty: a way to explicitly say what auth you want (including none) and a switch for multiple auth options (which could become a thing in the future if not a reality today).

Reply:
We can use an enum here. I guess the values would be `None` or `Certificate` (`ClientCertificate`?). Other modes, such as `ApiKey`, aren't planned for now.